src/backtrace.c

```diff
@@ -108,8 +108,10 @@ int backtrace_snapshot(int pid, int *tids, int *index, int nr_tids)
         return -1;
 
     for (i = 0; i < snap->num_threads; ++i) {
-        printf("--------------------  thread %d (%d)  --------------------\n",
+        printf("--------------------  thread %d (%d) (",
                (index != NULL ? index[i] : i+1), snap->tids[i]);
+        print_proc_comm(snap->tids[i]);
+        printf(")  --------------------\n");
```

Comment:
I agree.  I was not happy with how this looked, but I wanted to keep it a function call into proc and not have to store it in snapshot.

Do you have any suggestions?